doc/tutorials/12-constant_pH/si_units.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import scipy.constants as const
+
+# physical constants
+# kg atomic mass unit
+amu = const.physical_constants['atomic mass constant'][0]
+N_Av = const.N_A  # Avogadro 1/mol
+
+
+class si_units():
```

Comment:
please adhere to the naming convention, namely ```CamelCase``` for classes.

Reply:
@KaiSzuttor would you like it to be calles SIunits or SiUnits? I vote for the first option because it reads easier, although it is not quite camelcase.